.github/workflows/docker.yml

```diff
@@ -0,0 +1,48 @@
+name: Docker
+
+on:
+  pull_request:
+    branches: 
+      - master
+    types: [closed]
```

Comment:
If we push docker when a PR gets closed, how do we benchmark a PR as it's not in the dockerhub. Does it mean we can only benchmark a merged commit?

Reply:
maybe we should compile xline in every benchmark, otherwise it will generate a lot of useless tags on docker hub